### DISCUSSION TITLE
Johannus Reeds Off

### DISCUSSION THREAD

**Discussion by owenexplorerph** [GrandOrgue/grandorgue#2035] [Help]
How to program a sampleset created using GoODF to simulate the Johannus Reeds Off function?
The Johannus Reeds Off function functions similar to Anches de Appels of French organs, or ventils of some baroque organs.
Thanks for any assistance!

**Comment by larspalo** [comment]
Well, there are many ways to do a "Reeds Off" functionality but the exact implementation would depend on what you exactly want to happen when the different buttons are pushed.

A very simple solution that works exactly like "Appel d'Anches" is to just add a switch for the back-end reed stops that will be the "Reeds Off". Also each reed stop must have a separate switch with the same name as the stop that act as its activator and will be displayed instead of the back-end stop! The back-end stop could have a function of AND. The "Reeds Off" Switch could be displayed as inverted, have default to engaged on and "State when GC is activated" as ON.

Another way to achieve the exact same result is with an intermediate NOT switch in between the back-end stop and the "Reeds Off" switch. Then the display inverted, default to engaged and GCState change is not necessary anymore.

I attach a very small demonstration sample set below that contain two .organ files that use the different methods above.

[ReedsOff.zip](https://github.com/user-attachments/files/17684634/ReedsOff.zip)
